src/main/java/frc/robot/subsystems/flywheel/Flywheel.java

```diff
@@ -0,0 +1,74 @@
+package frc.robot.subsystems.flywheel;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.hardware.motor.ControllableMotor;
+import frc.utils.GBSubsystem;
+
+public class Flywheel extends GBSubsystem {
+
+	private final TopFlywheelComponents topFlywheelComponents;
+	private final BottomFlywheelComponents bottomFlywheelComponents;
+
+	private final ControllableMotor topMotor;
+	private final ControllableMotor bottomMotor;
+	private final FlywheelCommandsBuilder commandsBuilder;
+
+	public Flywheel(TopFlywheelComponents topFlywheelComponents, BottomFlywheelComponents bottomFlywheelComponents) {
+		super(FlywheelConstants.LOG_PATH);
+
+		this.topFlywheelComponents = topFlywheelComponents;
+		this.bottomFlywheelComponents = bottomFlywheelComponents;
+		this.topMotor = topFlywheelComponents.motor();
+		this.bottomMotor = bottomFlywheelComponents.motor();
+		this.commandsBuilder = new FlywheelCommandsBuilder(this);
+
+		updateInputs();
+	}
+
+	public double getTopMotorVoltage() {
+		return topFlywheelComponents.motorVoltageSignal().getLatestValue();
+	}
+
+	public double getBottomMotorVoltage() {
+		return bottomFlywheelComponents.motorVoltageSignal().getLatestValue();
+	}
```

Comment:
why needed?

Reply:
already told him to delete.. he thinks its needed